versions/development/SPEC.md

```diff
@@ -1088,49 +1095,54 @@ output {
 #### File Outputs
 
 Individual `File` outputs can be declared using a String path relative to the execution directory. For example:
-```
+
+```wdl
 output {
   File f = "created_file"
 }
 ```
 
 Notes:
 
- - When a file is output, the name and contents are considered part of the output but the local path is not.
- - If the specified file is a hard- or soft- link then it shall be resolved into a regular file for the WDL `File` output.
+* When a file is output, the name and contents are considered part of the output but the local path is not.
+* If the specified file is a hard- or soft- link then it shall be resolved into a regular file for the WDL `File` output.
 
 #### Directory Outputs
 
 A `Directory` can be declared as an output just like a `File`. For example:
-```
+
+```wdl
 output {
   Directory d = "created/directory"
 }
 ```
 
 Notes:
 
- - When a directory is output, the name and contents (including subdirectories) are considered part of the output but the path is not.
- - Any hard- or soft- links within the execution directory shall be resolved into separate, regular files in the WDL `Directory` value produced as the output.
+* When a directory is output, the name and contents (including subdirectories) are considered part of the output but the path is not.
+* Any hard- or soft- links within the execution directory shall be resolved into separate, regular files in the WDL `Directory` value produced as the output.
 
 ##### Soft link resolution example
 
 For example imagine a task which produces:
-```
+
+```sh
 dir/
  - a           # a file, 10 MB
  - b -> a      # a softlink to 'a'
 ```
 
 As a WDL directory this would manifest as:
-```
+
+```sh
```

Comment:
This is more `txt` than `sh`

Reply:
i agree but felt wanted the text after `#` to show up as comment